config/default/GUIDE.md

```diff
@@ -100,4 +100,4 @@ To add a condition card, add it to the "cards" section.
 To add a field inside an existing condition card, add it to the "fields" section of that card.
 You can remove the fields or condition cards that you don't want to show by deleting the whole code block belonging to that field/card.
 
-For more information: [Condition Cards documentation](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#cards)
+For more information: [Condition Cards documentation](http://localhost:1313/apps/features/contacts/#condition-cards)
```

Comment:
Looks like localhost should be replaced by `https://docs.communityhealthtoolkit.org`

Reply:
Hi @abbyad I have made that change and other changes in this commit: https://github.com/ronna/cht-core/commit/e7cbc33769ca04413ee7470af42bf05608e679bd how do I make another pull request

Reply:
As long as you make the commits in the same branch you don't need to make a further pull request. Seems like that's what you are doing, so it's all good!

Reply:
I still don't this updated to the docs website. I'll fix it here with a suggestion:

```suggestion
For more information: [Condition Cards documentation](http://docs.communityhealthtoolkit.org/apps/features/contacts/#condition-cards)
```